docs/discover/kuery.asciidoc

```diff
@@ -0,0 +1,114 @@
+[[kuery-query]]
+=== Kuery
+
+experimental[This functionality is experimental and may be changed or removed completely in a future release.]
+
+Kuery is a new query language built specifically for Kibana. It aims to simplify the search experience in Kibana
+and enable the creation of helpful features like auto-complete, seamless migration of saved searches, additional
+query types, and more. Kuery is a basic experience today but we're hard at work building these additional features on
+top of the foundation Kuery provides.
+
+Kuery queries are built with functions. Many functions take a field name as their first argument. Extremely common functions have shorthand notations.
```

Comment:
I was seeing how far I could get before someone told me to calm down with all the kuery queries 😄

Switched it to just be "Kueries"